### PR TITLE
Migrate CI to GitHub actions

### PR DESCRIPTION
**Pull request by browniebroke**
As Travis CI isn't running anymore. For security reasons, workflows added from a fork won't run on a PR, so you can't see them run here, but you can check the [results on my fork](https://github.com/browniebroke/django-datadog-logger/actions/workflows/ci.yml)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  push:
```
Comment (browniebroke) [inline .github/workflows/ci.yml:4]:
```suggestion
  push:
    branches:
      - master
```

**Comment by jacekbj** [comment]
@browniebroke What do you think about extending the version support to modern Python? I did that in my fork: https://github.com/jacekbj/django-dd-logger/blob/master/.github/workflows/pr-checks.yml#L35

**Comment by browniebroke** [comment]
Oh yes, I think that would be well needed! I was hoping to get this merged first and was planning to do it in a follow up PR.

**Comment by lnagel** [comment]
Closed in favor of @jacekbj -s version

**Comment by browniebroke** [comment]
This will need to be updated before release then: 

https://github.com/namespace-ee/django-datadog-logger/blob/c90df5b5a6ab8223bc81145839e1e8517485a704/setup.py#L36-L39

**Pull request closed without merging**